tools/build/tsconfig-base.json

```diff
@@ -1,35 +1,29 @@
 {
   "compilerOptions": {
-    "target": "es2017",
-    "lib": [
-      // Required for async iterators/generators:
-      "esnext.asynciterable",
-      // These are the defaults for "target": "es2017", but they have to be explicitly specified if we want to add anything to "lib":
-      "es2017",
-      "dom",
-      "dom.iterable",
-      "scripthost"
-    ],
-    "module": "commonjs",
-    "stripInternal": false,
+    "alwaysStrict": true,
     "declaration": true,
-    "strict": true,
+    "declarationMap": true,
+    "esModuleInterop": false,
+    "experimentalDecorators": true,
+    "forceConsistentCasingInFileNames": true,
+    "incremental": true,
+    "inlineSources": true,
+    "jsx": "react",
+    "module": "commonjs",
+    "moduleResolution": "node",
+    "noFallthroughCasesInSwitch": false,
     "noImplicitAny": true,
-    "strictNullChecks": true,
+    "noImplicitOverride": true,
+    "noImplicitReturns": true,
     "noImplicitThis": true,
-    "alwaysStrict": true,
     "noUnusedLocals": false,
     "noUnusedParameters": true,
-    "noImplicitReturns": true,
-    "noImplicitOverride": true,
-    "noFallthroughCasesInSwitch": false,
-    "experimentalDecorators": true,
+    "outDir": "lib/cjs",
```

Comment:
Did we want to change the default to be `lib/cjs` or leave it as `lib`?  If it's by default this way we could remove the explicit calls but I'm leaning more towards setting it to `lib` still and then anyone who overrides it need to go out of their way to make both esm and cjs

Reply:
Locally i reverted this change and have it pointing to lib, will push that out in a bit. Don't want to break any other consumers of this pkg